api/node/index.ts

```diff
@@ -356,6 +359,12 @@ export function loadFile(path: string) : Object {
     return slint_module;
 }
 
+export namespace Timer {
+    export function singleShot(duration: number, handler: () => void) {
+        napi.singleshotTimer(duration, handler)
+    }
+}
```

Comment:
Please add a comment here that this entire API should be removed. We should not need it, but we do until the event loop merge is ... merged.

Reply:
done